package.json

```diff
@@ -56,13 +62,18 @@
     "protractor": "~5.1.0",
     "rollup": "^0.41.6",
     "run-sequence": "^1.2.2",
-    "rxjs": "^5.1.0",
+    "rxjs": "^6.6.7",
     "ts-node": "~2.0.0",
     "tslint": "~4.5.0",
-    "typescript": "~2.2.0",
-    "zone.js": "^0.8.4"
+    "typescript": "~3.8.3",
+    "zone.js": "~0.10.3"
   },
   "engines": {
     "node": ">=6.0.0"
+  },
+  "dependencies": {
+    "@angular/cli": "^9.1.15",
+    "rxjs-compat": "^6.0.0-rc.0",
```

Comment:
is there a working stable version that isn't a release candidate?

Reply:
tested 6.6.7 appears to be working